lib/src/store.dart

```diff
@@ -85,8 +85,10 @@ class Store extends Disposable {
       throw new StateError('Store has been disposed');
     }
 
-    return _stream.listen(onData,
```

Comment:
This should be discussed.

I think a good rule of thumb is : "If code A creates a stream subscription, then codeA is responsible for cancelling it."

Thoughts ? @trentgrover-wf @maxwellpeterson-wf @evanweible-wf @sebastianmalysa-wf @jayudey-wf